### PR TITLE
solutions added test not running yet

### PR DESCRIPTION
**Pull request by tiggerman120**

### CODE REVIEW

**Comment by tiggerman120** [comment]
- [ x] Top-level README “Table of Contents” is updated
 - [ x] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [x ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [x ] Edge Case (if applicable/obvious)
 - [ ] README for this challenge is complete
     - [ x] Summary, Description, Approach & Efficiency, Solution
     - [ x] Link to code
     - [x ] Picture of whiteboard

File: javascript/challenges/linkedLists/llzip/ll-zip.test.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+const LL = require('../linked-list.js');
+
+describe('returns the head of two linked lists zipped together into one linked list', () => {
+  it('returns the head of a single linked list made from two linked lists', () => {
+    const list1 = new LL();
+    const list2 = new LL();
+    list1.insert(1).insert(2).insert(3);
+    list2.insert(4).insert(5).insert(6);
+    console.log(list1, list2);
+    expect(list1.head).toEqual(1);
```
Comment (adrianhuebner) [inline javascript/challenges/linkedLists/llzip/ll-zip.test.js:12]: This is off to a great start. I would suggest zipping the two linked lists, converting to a string and then checking that
